content/applications/sales/crm/pipeline/lost_opportunities.rst

```diff
@@ -1,27 +1,30 @@
-=========================
-Manage lost opportunities
-=========================
+==================
+Lost opportunities
+==================
 
 Not every opportunity results in a successful sale. To keep the pipeline up-to-date, *lost*
-opportunities need to be identified. Specifying why an opportunity was lost provides additional
-insight that can prove useful for future opportunities.
+opportunities need to be identified. Specifying why an opportunity was lost helps identify recurring
+issues, reveal coaching opportunities, and can assist with improving overall sales strategy.
+
+.. note::
+   :doc:`Merging lost opportunities <merge_similar>` with active ones will pull them back into the
+    pipeline.
```

Comment:
Remove this one space indent, which is causing the text to format as a [definition list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists).

Tip: enable the **Render Whitespace** setting in VS Code.
![Screenshot 2025-02-27 at 4 30 28 PM](https://github.com/user-attachments/assets/1137a0dc-14cf-49be-abf0-9fa2da462f8f)

Doing so allows you to see the number of space characters dont match ;)
![Screenshot 2025-02-27 at 4 33 32 PM](https://github.com/user-attachments/assets/e25159c4-f060-4fae-b7c1-d1e579617d0b)


```suggestion
   :doc:`Merging lost opportunities <merge_similar>` with active ones will pull them back into the
   pipeline.
```